stdlib/public/core/ArrayBuffer.swift

```diff
@@ -392,7 +342,7 @@ extension _ArrayBuffer {
       )
     } else {
       // ObjC arrays do their own subscript checking.
-      element = _nonNative.objectAt(i)
+      element = _nonNative[i]
```

Comment:
Nitpick: This was probably written this way because `-objectAtIndexedSubscript:` is just going to call `-objectAtIndex:`, and you might as well save that one objc_msgSend.

Reply:
This does not call `-objectAtIndexedSubscript:`: `_nonNative` now returns a `_CocoaArrayWrapper`, which is a Swift struct.  Its opaque subscript calls `objectAt(_:)`, which is the shadow protocol's pet name for `-objectAtIndex:` a.k.a. `object(at:)`

This does add an extra static function call compared to the original, but those are pretty cheap, and this is on a slow path anyway, so it shouldn't matter much.

Reply:
Aha, my bad for not taking the whole patch at once. Thanks for the explanation.